rosetta_matrices.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import pandas as pd
```

Comment:
You put this new version of the function in its own file. However, you should instead replace the previous version of your code that's in the `rosetta.py` file. Please move the code there, then delete this file